2_form.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>My first form here</title>
+</head>
+<body>
+
+  <h1>Payment Form</h1>
+  <h4>Requierd Fields are followed by .*.</h4>
+  <p style="font-size:30px">Contact Information</p> <!--Using this to write here in large size-->
+  <fieldset> 
```

Comment:
We use fieldset tag within form tag

Reply:
Okay so I'll write form tag before fieldset tag, right?

Reply:
correct